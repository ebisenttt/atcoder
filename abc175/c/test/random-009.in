-195732710689846 973784499358144 25853134137841
