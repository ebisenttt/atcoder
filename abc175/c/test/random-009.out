11092362412882
